opendap_protocol/protocol.py

```diff
@@ -552,3 +552,12 @@ def meets_constraint(constraint_expr, data_path):
             return True
 
     return False
+
+
+def set_dask_encoding_chunk_size(chunk_size: int):
```

Comment:
do we need this method? the class attributes could be changed just by importing.